.github/workflows/test-package.yml

```diff
@@ -1,4 +1,4 @@
-name: Dart CI
+name: ci
```

Comment:
Just FYI, this is the name that will show in the markdown badge.

Reply:
🤷  – do you like the other name more?

Reply:
just updated the action version instead 😄

Reply:
`Dart`? No particular preference, just fyi, that the markdown badge's name would have gotten short :)